src/olympia/blocklist/admin.py

```diff
@@ -43,6 +44,37 @@ def _get_version_choices(block, field_name):
     return choices
 
 
+class BlocklistSubmissionStateFilter(admin.SimpleListFilter):
```

Comment:
Wow, that's a lot of code just to switch from having a filter without a default to having a filter with a default!

Reply:
yeah, I know.  I tried to write it as generically as possible so it could be reused but it'd be another issue to see if some of the other Admin classes could be rewritten to use it too (e.g. the scanners admin uses a number of ListFilter classes - but they're doing other things as well changing the default)